### ISSUE TITLE
Cannot reproduce fine-tuned model paper results

### ISSUE THREAD

**Issue by moore269** [ShayanTalaei/CHESS#24]
On the sub-sampled dev set, I am able to get 51% accuracy using run_main_ir_ss_cg.sh. I believe this is close to the accuracies the paper claimed. However, when I switch to the fine-tuned model that is suggested in model_deployment.txt (AI4DS/NL2SQL_DeepSeek_33B), I only get 37.4% accuracy. 

I am able to deploy the model successfully on a100 gpu and successfully setup a different config file that copies CHESS_IR_SS_CG.yaml and replaces only the candidate generation part with the deepseek model. I am using the template_generate_candidate_finetuned.txt as well as the generated_candidate_finetuned parser.

Am I using the correct fine-tuned model? Are there other settings I need to be aware of to replicate the paper's results?

**Comment by MohammadrezaPourreza** [comment]
Hi @moore269,

Thank you so much for your interest in our work! Since the code has undergone significant changes following our re-submission to BIRD, could you please switch to the chess-v1 branch and test the fine-tuned model again?

We used the same model for the BIRD holdout test set, so it should work as expected.

Thank you for your understanding and support!

Best regards,
Mohammadreza

**Comment by moore269** [comment]
Thanks for the suggestion @MohammadrezaPourreza. I tried your suggestion, and it improved the fine-tuned model results to 46.25%. However, this is still lower than the 51% I was getting before using the standard gpt calls. I expect the fine-tuned model to perform better and closer to the 60ish% that the paper reports. Could you look into this?

Also, I don't mind switching to a newer fine-tuned model you may have if its easier for you to just add new config files to your latest main branch.

**Comment by MohammadrezaPourreza** [comment]
Actually this is a bit strange and unexpected as we shared the same model with the BIRD authors and the reported results on the leaderboard is obtained with the same model. In order to better locate the issue can you please send your results directory to my email? pourreza@ualberta.ca

**Comment by Kirushikesh** [comment]
Hello @MohammadrezaPourreza @ShayanTalaei is it possible or any plans to share the finetuned DeepSeek 34B LLM used in the newer version of the paper?
